server/controllers/match.js

```diff
@@ -18,11 +18,11 @@ const joinMatch = (req, res) => {
     {_id: matchID},
     {$push: {'userIDs': {id}}},
     {new: true}
-  ).then((meg) => {
-    console.log(meg);
+  ).then(() => {
+    res.status(200).jason({status: 'Joining match'});
```

Comment:
jason should be changed to json here.

Reply:
a good way to test if your backend is working properly and sending the correct response is by using a tool like postman

Reply:
Error is modified.